packages/boxel-ui/test-app/tests/acceptance/index-test.ts

```diff
@@ -10,14 +10,19 @@ module('Acceptance | Docs', function (hooks) {
     await visit('/');
     assert.dom('h2.FreestyleUsage-name').containsText('Accordion');
     // Only audit usage-preview examples
-    await a11yAudit({
-      include: ['.FreestyleUsage-preview'],
-      // https://github.com/dequelabs/axe-core/issues/3082
-      // turn off the rule for aria-allowed-role for now until ember-a11y-testing is updated with bugfix from axe-core
-      rules: {
-        'aria-allowed-role': { enabled: false },
+    await a11yAudit(
+      {
+        include: ['.FreestyleUsage-preview'],
       },
-    });
+      {
+        // https://github.com/dequelabs/axe-core/issues/3082
+        // turn off the rule for aria-allowed-role for now until ember-a11y-testing is updated with bugfix from axe-core
+        rules: {
+          'aria-allowed-role': { enabled: false },
+          'color-contrast': { enabled: false },
```

Comment:
I have to disabled this rule because with that rule, button with white background and teal color isn't allowed.

<img width="423" alt="Screen Shot 2023-12-05 at 14 57 41" src="https://github.com/cardstack/boxel/assets/12637010/c426beb8-62cd-41b2-8d21-e30520c5d43e">